src/components/ui/coaching-sessions/coaching-notes.tsx

```diff
@@ -258,22 +306,33 @@ const createLinkClickHandler = () => (_view: unknown, event: Event) => {
   return false;
 };
 
-const isShiftClickOnLink = (target: HTMLElement, event: MouseEvent): boolean => {
+const isShiftClickOnLink = (
+  target: HTMLElement,
+  event: MouseEvent
+): boolean => {
   return !!(
     (target.tagName === "A" || target.parentElement?.tagName === "A") &&
     event.shiftKey
   );
 };
 
 const openLinkInNewTab = (target: HTMLElement) => {
-  const href = target.getAttribute("href") || target.parentElement?.getAttribute("href");
+  const href =
+    target.getAttribute("href") || target.parentElement?.getAttribute("href");
   if (href) {
     window.open(href, "_blank")?.focus();
   }
 };
 
 const handleEditorContentError = (error: unknown) => {
-  console.error('Editor content error:', error);
+  console.error("Editor content error:", error);
+
+  // Show user-friendly error message via toast
+  if (error instanceof TypeError && error.message.includes("Expected table token")) {
+    toast.error("Unable to paste table. The table format may be invalid.");
```

Comment:
Matching on actual error output strings is usually fairly brittle in my own experience. This one seems pretty specific as well since, I would assume, user's will be using more than just tables. I like the generic toast message below. That, to me, would handle all cases well.

Reply:
Addressed in 6fb18fc